life_expectancy/tests/test_main.py

```diff
@@ -7,12 +7,26 @@
 from life_expectancy.main import main
 
 
-def test_main(pt_life_expectancy_expected):
-    """Run the `main` function and compare the output to the expected output"""
+def test_main_csv(pt_life_expectancy_expected):
```

Comment:
The behavior of the main function should not be agnostic to file types? assuming that you have also tested the cleaning function for both file types...

Reply:
Resolved